### ISSUE TITLE
Please add simplelogin.io, rambler.ru, emailnator.com, noip.com, seznam.cz, wp.pl, skiff.com, tempmaili.com, proton.me, courvix.com, tutanota.com, 5ymails.com, internxt.com, tempmail.email, emailondeck.com, tempor.email, fakesmail.com, tempimail.org, temp-mailbox.com, temp-mail.us, 10minemail.com, hour.email, cloudtempmail.com, tmail.ai, zemail.me, tempumail.com, emailfake.com, o2.pl, and addy.io

### ISSUE THREAD

**Issue by GalacticHypernova** [7c/fakefilter#73]
## simplelogin.io:
The current domains according to what I can see are:
slmails.com
simplelogin.com
aleeas.com
slmail.me
silomails.com
8shield.net
8alias.com
dralias.com
simplelogin.fr

Their entire project is also open source so there may be more domains hidden within the source (maybe even the domain generation process)
https://github.com/simple-login/
EDIT: I dug around their API, and here's a resource to obtain all possible domain (without custom domains that require valid MX records) (you must be logged in to the app or else it says wrong api key):
https://app.simplelogin.io/api/v2/setting/domains
![image](https://github.com/7c/fakefilter/assets/92037085/a4ead6fa-1006-44d6-8807-143e5231d295)

## addy.io:
[possible subdomain].anonaddy.me
[possible subdomain].anonaddy.com
Also open source:
https://github.com/anonaddy/anonaddy
EDIT: Dug around their API as well, it seems as though they need a user to be logged in:
https://app.addy.io/api/v1/domains but would require a bit of digging becausse they need authentication which I don't really know how to integrate with fakefilter considering I don't know what technology you use, so I'll leave that to you.

There is however another endpoint that displays the domain options, which is as follows:
https://app.addy.io/api/v1/domain-options
Returns:
![image](https://github.com/7c/fakefilter/assets/92037085/0fdb72c0-cc8c-4833-a099-b06f35cb412d)

The data is the array of the possible domains a logged in user can use. The domains is used for custom, additional domains that like simplelogin require valid records.

## noip.com:
noip.com
ddns.net
ddnsking.com
3utilities.com
bounceme.net
freedynamicdns.net
freedynamicdns.org
gotdns.ch
hopto.org
myddns.me
myftp.biz
myftp.org
myvnc.com
onthewifi.com
redirectme.net
servebeer.com
serveblog.net
servecounterstrike.com
serveftp.com
servegame.com
servehalflife.com
servehttp.com
serveirc.com
serveminecraft.net
servemp3.com
servepics.com
servequake.com
sytes.net
viewdns.net
webhop.me
zapto.org
access.ly
blogsyte.com
brasilia.me
cable-modem.org
ciscofreak.com
collegefan.org
couchpotatofries.org
damnserver.com
ddns.me
ditchyourip.com
dnsfor.me
dnsiskinky.com
dvrcam.info
dynns.com
eating-organic.net
fantasyleague.cc
geekgalaxy.com
golffan.us
health-carereform.com
homesecuritymac.com
homesecuritypc.com
hosthampster.com
hopto.me
ilovecollege.info
loginto.me
mlbfan.org
mmafan.biz
myactivedirectory.com
mydissent.net
myeffect.net
mymediapc.net
mypsx.net
mysecuritycamera.com
mysecuritycamera.net
mysecuritycamera.org
net-freaks.com
nflfan.org
nhlfan.net
pgafan.net
point2this.com
pointto.us
privatizehealthinsurance.net
quicksytes.com
read-books.org
securitytactics.com
serveexchange.com
servehumour.com
servep2p.com
servesarcasm.com
stufftoread.com
ufcfan.org
unusualperson.com
workisboring.com
Appears to have a paid API access, will try to dig deeper to find the endpoint.
EDIT: Dug around their API, the domain list is retrievable through the following code (it's an inline script that exists at https://my.noip.com only, from my research):
```ts
const element=document.getElementById("app_config")
const json=JSON.parse(element.textContent)
const domains=json.domains
```
![image](https://github.com/7c/fakefilter/assets/92037085/12d87d19-bed6-460c-bad9-19a87af8e547)

Or in text form:
```json
{
    "free": [
        "ddns.net",
        "ddnsking.com",
        "3utilities.com",
        "bounceme.net",
        "freedynamicdns.net",
        "freedynamicdns.org",
        "gotdns.ch",
        "hopto.org",
        "myddns.me",
        "myftp.biz",
        "myftp.org",
        "myvnc.com",
        "onthewifi.com",
        "redirectme.net",
        "servebeer.com",
        "serveblog.net",
        "servecounterstrike.com",
        "serveftp.com",
        "servegame.com",
        "servehalflife.com",
        "servehttp.com",
        "serveirc.com",
        "serveminecraft.net",
        "servemp3.com",
        "servepics.com",
        "servequake.com",
        "sytes.net",
        "viewdns.net",
        "webhop.me",
        "zapto.org"
    ],
    "enhanced": [
        "access.ly",
        "blogsyte.com",
        "brasilia.me",
        "cable-modem.org",
        "ciscofreak.com",
        "collegefan.org",
        "couchpotatofries.org",
        "damnserver.com",
        "ddns.me",
        "ditchyourip.com",
        "dnsfor.me",
        "dnsiskinky.com",
        "dvrcam.info",
        "dynns.com",
        "eating-organic.net",
        "fantasyleague.cc",
        "geekgalaxy.com",
        "golffan.us",
        "health-carereform.com",
        "homesecuritymac.com",
        "homesecuritypc.com",
        "hosthampster.com",
        "hopto.me",
        "ilovecollege.info",
        "loginto.me",
        "mlbfan.org",
        "mmafan.biz",
        "myactivedirectory.com",
        "mydissent.net",
        "myeffect.net",
        "mymediapc.net",
        "mypsx.net",
        "mysecuritycamera.com",
        "mysecuritycamera.net",
        "mysecuritycamera.org",
        "net-freaks.com",
        "nflfan.org",
        "nhlfan.net",
        "pgafan.net",
        "point2this.com",
        "pointto.us",
        "privatizehealthinsurance.net",
        "quicksytes.com",
        "read-books.org",
        "securitytactics.com",
        "serveexchange.com",
        "servehumour.com",
        "servep2p.com",
        "servesarcasm.com",
        "stufftoread.com",
        "ufcfan.org",
        "unusualperson.com",
        "workisboring.com"
    ],
    "affiliate": []
}
```

## emailnator.com:
smartnator.com
femailtor.com
Randomly generated Gmail accounts
googlemail.com 
mydefipet.live
tmpmailtor.com
psnator.com
Has a freemium API for bulk domains:
```ts
const axios = require('axios');

const options = {
  method: 'POST',
  url: 'https://gmailnator.p.rapidapi.com/bulk-emails',
  headers: {
    'content-type': 'application/json',
    'X-RapidAPI-Key': 'b0d5becc39msh5d66ccd02a4f40dp16f7a9jsn4315d6fbc1dc',
    'X-RapidAPI-Host': 'gmailnator.p.rapidapi.com'
  },
  data: {
    limit: 20,
    options: [1, 2, 3]
  }
};

try {
	const response = await axios.request(options);
	console.log(response.data);
} catch (error) {
	console.error(error);
}
```
Something interesting I learnt about the generated gmail accounts is that they either contain + or an excessive amount of dots, 2 very odd patterns in real, legitimate email accounts. I was able to reliably detect at least 3 or 4 dots in the email itself when it's using the dotted variation.

## rambler.ru:
rambler.ru
myrambler.ru
autorambler.ru
rambler.ua
ro.ru

This site is a semi legit news site, but the mail service is used for temp emails.

Their API endpoint: 
https://id.rambler.ru/api/v3/legacy/Rambler::Id::get_available_domains

It appears to have very odd authentication type middleware, it can only be used from a **[certain part](https://id.rambler.ru/login-20/mail-registration?rname=mail&theme=&session=false&back=https%3A%2F%2Fmail.rambler.ru%2F&param=embed&iframeOrigin=https%3A%2F%2Fmail.rambler.ru)** of their website with the following fetch:
```ts
await fetch("https://id.rambler.ru/api/v3/legacy/Rambler::Id::get_available_domains", {
  "headers": {
    "accept": "*/*",
    "accept-language": "he,en;q=0.9,en-GB;q=0.8,en-US;q=0.7",
    "cache-control": "no-cache",
    "content-type": "application/json",
    "pragma": "no-cache",
    "sec-ch-ua": "\"Microsoft Edge\";v=\"117\", \"Not;A=Brand\";v=\"8\", \"Chromium\";v=\"117\"",
    "sec-ch-ua-mobile": "?0",
    "sec-ch-ua-platform": "\"Windows\"",
    "sec-fetch-dest": "empty",
    "sec-fetch-mode": "cors",
    "sec-fetch-site": "same-origin",
    "x-client-request-id": "ridduMw2OekRXLeVWwTD"
  },
  "referrer": "https://id.rambler.ru/login-20/mail-registration?rname=mail&theme=&session=false&back=https%3A%2F%2Fmail.rambler.ru%2F&param=embed&iframeOrigin=https%3A%2F%2Fmail.rambler.ru",
  "referrerPolicy": "strict-origin-when-cross-origin",
  "body": "{\"id\":\"ridduMw2OekRXLeVWwTD\",\"params\":{}}",
  "method": "POST",
  "mode": "cors",
  "credentials": "include"
})
```
And returns the following:
![image](https://github.com/7c/fakefilter/assets/92037085/7a20d8b8-42fc-4f1d-8b84-6c128687641e)

## seznam.cz
seznam.cz

## wp.pl
wp.pl

## tempmaili.com
hopeon.xyz
munik.edu.pl
(Looking for the API for that)

## skiff.com
skiff.com 
[possible subdomain].maskmy.id
This should be a wildcard as there are infinite amount of subdomains all under maskmy.id

## proton.me
proton.me
protonmail.com

## courvix.com
API endpoint: https://courvix.com/api/emails
Returns all of the possible domains (below is a screenshot of the partial response):
![image](https://github.com/7c/fakefilter/assets/92037085/cb5e36ec-ebaf-4e07-a053-9bc3f4c10511)

## tutanota.com
tutanota.com
tutanota.de
tutamail.com
tuta.io
keemail.me
tuta.com
A way to automate this process would be to go to https://app.tuta.com/common-min-b91bbf95.js and take the `e("bW",["tutanota.com","tutanota.de","tutamail.com","tuta.io","keemail.me","tuta.com"])` part (whether it be through regex or a different method). The [**source map**](https://app.tuta.com/common-min-b91bbf95.js.map) for this has it as an export: `export const TUTANOTA_MAIL_ADDRESS_DOMAINS = [\"tutanota.com\", \"tutanota.de\", \"tutamail.com\", \"tuta.io\", \"keemail.me\", \"tuta.com\"]`

## 5ymail.com
5ymails.com
5ymail.com
5ymail.net
5ymail.de
e9mail.com
e9hosting.com
5ymail.me

## internxt.com
theeyeoftruth.com
beaconmessenger.com
lamasticots.com

They are open source so I'm still looking into where their API is

## tempmail.email
The endpoint: https://tempmail.email/api/domains.alternative.config.json
Returns all available domains:
![image](https://github.com/7c/fakefilter/assets/92037085/d39fea87-5a5d-4c3f-985b-f5026b620d05)
They have another endpoints: https://tempmail.email/api/domains.config.json
Which returns more domains:
![image](https://github.com/7c/fakefilter/assets/92037085/44ad19c8-0c81-4348-a923-9df400be3f7c)

## emailondeck.com
sloveniakm.com appears to be the only domain, has a PRO api endpoint to list all available domains that requires a token: 
https://api.emailondeck.com/api.php?token=[TOKEN_HERE]&act=list_email_domains&order=date_desc

## tempor.email
scandicdeals25.com
nurumassager.com
varaunited.in
whitworthknifecompany.com
hatberkshire.com
moreview.xyz
nmemail.xyz
aconnectioninc.com
bendnsend.com
waleskfb.com
lamedicalbilling.com
fabtours.live
klearlogistics.com
stragedycd.com
igeekmagz.pw
91sedh.xyz
besttimenews.xyz
secure-mail.cc
dromancehu.com
wremail.xyz
ohioflyfishinguides.com
affordableroofcare.com
redfaunstudio.com
aeshopshop.xyz
uqkemail.xyz
elmcreekcoop.com
linseyalexander.com
otratransportation.com
z1p.biz
wremail.top
stivendigital.club
gssfire.com
asifboot.com
pliqya.xyz
tmailcloud.net
andersonelectricnw.com
3mail.rocks
pwruttz.com
customequipmentstore.com
partmed.net
csderf.xyz
mylandjet.com
wuupr.com
ivyplayers.com
bacinj.com
volunteerindustries.com
liquidlogisticsmanagement.com
tlcemail.xyz
colourmedigital.com
childrenofthesyrianwar.com
jgwinindia.com
mailrock.biz
unicomti.com
buzztrucking.com
wellnessconnecthc.com
amazinggift.life
meibokele.com
mailpoly.xyz
bukutututul.xyz
syonacosmetics.com
abunasser.site
efundpro.com
360wellnessuk.com
ambientiusa.com
btcmod.com
twichzhuce.com
straightflightgolf.com
strategysuperb.com
leafrelief.org
uniaotrafego.com
garoofinginc.com
nmemail.top
acampadaparis.com
gofsrhr.com
skxemail.com
pekanrabu.biz
thelubot.site
tesqwiklabsss.shop
eellee.org
jwpemail.eu
apexhearthealth.com
yandexmailserv.com
They have an API which I'm trying to explore right now, but for the time being I had to write this all down

## fakesmail.com
stacktix.one
votexrise.com
otelekom.live
xtoox.com
profunivers.site
enoiv.pro
fugmi.xyz

## tempimail.org
iemail.fun
wemail.pics
loveblog.help

## temp-mailbox.com
codjobs.com
jocdk.com
mvpve.com
page.edu.pl
sdkjob.com
codvip.net

## temp-mail.us
fosil.pro
rootfest.net
polyfaust.com
cartelera.org
hostcalls.com

## 10minemail.com
jucatyo.com
eazenity.com
mkurg.com
othao.com
newnime.com
mainmile.com
I know this website is already monitored, however I couldn't find these specific domains in the list
It has a freemium API just like emailnator:
```ts
const axios = require('axios');

const options = {
  method: 'GET',
  url: 'https://privatix-temp-mail-v1.p.rapidapi.com/request/domains/',
  headers: {
    'X-RapidAPI-Key': 'b0d5becc39msh5d66ccd02a4f40dp16f7a9jsn4315d6fbc1dc',
    'X-RapidAPI-Host': 'privatix-temp-mail-v1.p.rapidapi.com'
  }
};
try {
	const response = await axios.request(options);
	console.log(response.data);
} catch (error) {
	console.error(error);
}
```

## hour.email
mynamejewel.com

## cloudtempmail.com
bigddns.net
4save.net
taxibmt.com
videotoptop.com
ipxwan.com
email84.com
soc123.net
nhmvn.com
nickmxh.com
cloudtempmail.net
choigi.com
coffeejadore.com
bigddns.org
acc1s.com
dulich84.com
pingddns.org
emailtik.com
huongdanfb.com
tatadidi.com
tubeemail.com
5semail.com
nextsuns.com
tinpho.com
embekhoe.com
mp3oxi.com
n-h-m.com
mikfarm.com
nuoifb.com
voxinh.net
vinakop.com
storebanme.com
email-68.com
emailcoffeehouse.com
camera47.net
mikrotikvn.com
kenhbanme.com
muadaingan.com
pingddns.com
xehop.org
connho.net
vinaemail.com
fastddns.net
vncctv.net
nick-ao.com
bigddns.com
emailracc.com
chamconnho.com
coffeeazzan.com
datadudi.com
mikrotikvietnam.com
libinit.com
taxibmt.net
cloneemail.com
giodaingan.com
vobau.net
diemhenvn.com
mikrotikx.com
vinakoop.com
123clone.com

## tmail.ai
happy9toy.com
leechchannel.com
likemovie.net
pertera.com
Couldn't get more because their firewall is broken but I would assume some shared domains would be in cloudtempmail.com

## zemail.me
solariss.me
blocki.tech
thimble.email
codda.me
astern.live
(this one will be hard to distinguish) gmail.com

## tempumail.com
umail.edu.pl
tempumail.online
ezblog.co

## emailfake.com
proceedwky.com
jdefiningqt.com
legal-research-investigation.com
kocheme.com
varaunited.in
avobitekc.com
epppl.com
jnckteam.eu
wantwp.com
guesthousenation.com
cabovertrans.com
nyfinestbarbershop.com
bromeil.com
strategysuperb.com
sikatan.co
7814445.com
eligibilitysolutions.com
discardmail.ninja
haqoci.com
shiningblogpro.com
easy-mail.top
rumahcloudindonesia.online
salonkarma.club
mentornkc.com
gspousea.com
uenglandrn.com
senduvu.com
affliatemagz.com
mywayzs.com
And any other domain with valid records
This one is a bit harder to collect all domains from, I have noticed their script however I am still trying to figure out a way to use it to obtain all possible domains. They're using jQuery and typeahead so that may be useful information.

## o2.pl
o2.pl

**Comment by L0calH0st80** [comment]
I was going to report the same but looks like you did already.

**Comment by GalacticHypernova** [comment]
Also, just in case you haven't seen my response at  #69 , please let me know where it would be convenient to talk privately about a potentially very sensitive update idea.

**Comment by beeyev** [comment]
`rambler.ru` is not a temp mail service, it's a news portal and search engine.

**Comment by GalacticHypernova** [comment]
@beeyev Well, yes, but the mail part is very much used for temp emails, which is also what I explained in my message 🙂

**Comment by Nikola-Test-for-Proton** [comment]
Hello,

My name is Nikola and I work in Proton's (and SimpleLogin's) Anti-Abuse team. 

Just to make sure that I understand the discussion here, is this a request to add the SL domains mentioned above to a list of disposable (or temporary) emails? Keep in mind that email aliases are not temporary, but rather permanent. 

We also take Anti-Abuse seriously and have different mechanisms for ensuring against bulk creation of accounts (be it on-platform or for third parties), sending spam or unwanted emails, and the like. 

Looking forward to hearing back from you.

**Comment by GalacticHypernova** [comment]
@Nikola-Test-for-Proton Yes, you got it right. This is a request to add SL, among others, to the list as it is indeed a fake email provider. I am pleased to hear that you take anti-abuse seriously, but I've seen spikes in temporary email usage from SL's domains in various places online, which led to major abuse, so it is very much problematic, which is why I would appreciate if you could implement some anti-abuse mechanisms. While your email aliases are permanent, there is nothing stopping a malicious party from simply deleting the alias after the goal for which the alias was created has been completed and create a new one. Therefore, SL is, by all means, a temp email provider. 

Looking forward to hearing back from you (preferably without another copy pasted comment from the other issues you have opened, as I would much rather talk to a person than to a copy paste machine)

**Comment by Nikola-Test-for-Proton** [comment]
Hello,

Thank you for following up. 

Please note that there are only so many ways to say what we did in similar threads we've recently created, namely expressing our interest in understanding why our domains are being listed as disposable (1) and extending an offer to the maintainer(s) to work with us in order to improve our anti-abuse mechanisms going forward (2). Apologies if you feel otherwise. 

Keep in mind that we do have systems in place for dealing with abuse on all our services and SimpleLogin is no different. If you would be so kind as to share details regarding the cases you've mentioned, we would greatly appreciate it so that our team can then analyze things more deeply and take measures to attune our procedures accordingly. 

As a side note, could you clarify why you also consider the Proton Mail domains to be 'fake'?

**Comment by GalacticHypernova** [comment]
@Nikola-Test-for-Proton I appreciate your response. I'd gladly elaborate. However, I think this discussion is better to be had privately. Please let me know where you would like for us to continue this, so as to not flood this issue with a conversation most don't have any use in.

**Comment by Nikola-Test-for-Proton** [comment]
Sure, feel free to reach out by sending a message at antiabuse@proton.me.

**Comment by GalacticHypernova** [comment]
@Nikola-Test-for-Proton Are you willing to settle for a more convenient method of communication, like Discord?

**Comment by Nikola-Test-for-Proton** [comment]
That can work as well. Please provide your username when you get a chance so we can send you a message.

**Comment by GalacticHypernova** [comment]
@Nikola-Test-for-Proton sure thing! galactichypernova

**Comment by Ufyfughx** [comment]
In this case you should block:

(no verification)
skiff.com
tutanota.com
tutanota.de
tutamail.com
tuta.io
keemail.me

anonaddy.com
anonaddy.me

anonaddy.com and anonaddy.me are the same of SimpleLogin.

If you are blocking such services then block all. Or unblock SimpleLogin because now it is kind of discrimination.

**Comment by Ufyfughx** [comment]
@GalacticHypernova

**Comment by GalacticHypernova** [comment]
Wish I could thank you for the addition, but looks like you got deleted

**Comment by ctdyfyv** [comment]
Add 5ymail.com too. It can receive mails, not only send

**Comment by Hbvgvcbhc** [comment]
Also block *.maskmy.id

[Screenshot 1](https://github.com/7c/fakefilter/assets/149825198/aeddc1ae-00be-4d04-846a-13f00e771cdc)
[Screenshot 2](https://github.com/7c/fakefilter/assets/149825198/a5c59d69-35ce-47f0-b49b-1d6716757379)
[Screenshot 3](https://github.com/7c/fakefilter/assets/149825198/3a427d4d-6222-497a-bd65-c9145bb40f6c)
[Screenshot 4](https://github.com/7c/fakefilter/assets/149825198/8cea9901-e74f-4ac5-b8f4-127ed91dd227)

**Comment by GalacticHypernova** [comment]
Thanks ghosts! Added!

**Comment by Hchgughcyfyg** [comment]
https://github.com/7c/fakefilter/issues/73#issuecomment-1830118249 forgotten `protonmail.com` domain

**Comment by GalacticHypernova** [comment]
@7c do you know where all of these ghost accounts come from by any chance? First time I'm seeing such a big amount of them..

**Comment by Gchcuchxgchc** [comment]
> them.

It’s not many accounts. It is all mine.

GitHub rejects my privacy mail so I delete accounts not to be tracked by Microsoft.

It is still me.

By the way, why to include Proton but not Gmail or Outlook? There is simmilar registration without any confirmation.

**Comment by Gchcuchxgchc** [comment]
Proton protects my privacy. Especially against some bad people. I need to use encrypted services for my privacy. Your request is negogate all my efforts to stay secure and private:(

I hope you will reconsider your decision

**Comment by GalacticHypernova** [comment]
Honestly, the more these one time accounts try to convince me to remove protonmail and simplelogin, the more I see how much it's needed to block it. It's like the marketing team is desperately trying to keep their services from being rightfully flagged and it just makes me want to block them even more. I do hope they won't cloud your judgement @7c , as these services are used as temp email services by all definitions. If you have any questions you may always ask me from the conversation we have started :)

**Comment by GalacticHypernova** [comment]
> By the way, why to include Proton but not Gmail or Outlook?

By that logic, why not include every mail service in the world? Of course there's some level of anonymity in any email registration service, you can't physically verify with your body or something. It's about the reputation and the amount of anonymity required. I'm honestly surprised and a little disappointed you even implied it as with all due respect, it's very much obvious.

**Comment by NiaTheCat** [comment]
I guess since I don't want to use a Google or a Microsoft product, that I'm not allowed to use sites. Cool.

**Comment by Retold3202** [comment]
Proton, Tutanota, and Skiff are legitimate email providers. This filter claims to "sweep temporary and fake email addresses from registrations", and I don't think either of those 3 providers meet that definition at all. They're legitimate email services with a good reputation.

If they should be blocked, you should really consider blocking Gmail, Outlook, Yahoo, etc. as well. I'd be interested to hear any compelling arguments as to how those providers are allowed and justified, but these aren't? The only difference is Proton, Tutanota, and Skiff value privacy and their users. People should have the choice, and I fail to see how blocking them would meaningfully reduce abuse of registrations in really any way. 

Please respect others' choice of email providers and people who value privacy. All this will do is alienate people from using websites that rely on this filter, while accomplishing nothing of value.

**Comment by D4llo** [comment]
> @Nikola-Test-for-Proton I appreciate your response. I'd gladly elaborate. However, I think this discussion is better to be had privately. Please let me know where you would like for us to continue this, so as to not flood this issue with a conversation most won't have any use in.

I want to learn more about this can you post everything here? At least a summary?

**Comment by cristian64** [comment]
I don't know the vast majority of the blocked domains, but I do use `protonmail.com`, and it makes no sense that it would be blocked while others like `gmail.com` are not.

This is a detrimental change for `fakefilter`; please reconsider.

**Comment by TheUncannyObserver** [comment]
This PR is abusive and should be denied, especially since the author provides no empirical proof whatsoever that these services are responsible for any sort of abuse, widespread or not.

**Comment by yrahul3910** [comment]
God, what a dogshit idea to include Proton/Tutanota/etc. Great job making the web less private.

**Comment by scrufdawg** [comment]
This "issue" should really be closed with prejudice.

**Comment by GalacticHypernova** [comment]
What all of you fail to realize is that it's literally the point of the project. To filter those "emails" that value privacy. Just like everyone has the right to use those privacy prioritized emails, website operators have a right to know. That is a 1:1 copy of the description of the project.

**Comment by taiiat0** [comment]
> What all of you fail to realize is that it's literally the point of the project. To filter those "emails" that value privacy. Just like everyone has the right to use those privacy prioritized emails, website operators have a right to know. That is a 1:1 copy of the description of the project.

"literally the point of the Project" - i think you have not as of yet Read the single Paragaph on the Readme for this Project that tells you exactly what the point of the Project is, which is not what you say it is.

**Comment by GalacticHypernova** [comment]
> @GalacticHypernova That is absolutely not a "1:1 copy of the description of the project". From the README,
> 
> > The intention of this project is to sweep temporary and fake email addresses from registrations. We collect fake email providers and their domains and want to publish them here in different formats. We want that site owners use this repo as reliable source to find out all fake/temp provider registration attempts.
> 
> Proton/Tutatnota/Skiff are neither fake email providers, nor temp providers. If anything, like a commenter pointed out above, GMail almost certainly has the most temp emails.

And now read the rest...
"but also service providers do have the right to know about this fact."

**Comment by yrahul3910** [comment]
> And now read the rest... "but also service providers do have the right to know about this fact."

Except that it isn't a fact. These are simply NOT temp email providers.

> Do you really want to tell me that the ability to create infinite aliases and mailboxes with no repercussions whatsoever is not considered a temp email?

Sigh. You can do the same with GMail. Or Outlook. Or _any_ email service. By the same logic, someone can use Python to DDoS a server. Do you really want to tell me that the ability to DDoS any server in the world is not considered a DDoS tool?

**Comment by GalacticHypernova** [comment]
> So you'd have no problem adding `privaterelay.appleid.com` to your list, then :)

I have not seen that service beforehand, but I can look into it. If it's actually just like SL, then yea, sure. Thanks for notifying me about it! I appreciate it.

**Comment by Zoarial94** [comment]
Hello, I'm not affiliated with anyone or any company, but I do pay for Proton's services. 

I switched over to Proton Mail a few years ago and have never looked back. Proton Mail should absolutely not be on this list. There isn't anything which makes the service temporary. Email address can't be re-used or recycled. Paid accounts can generate a small number of alternate emails, but those are limited in number and are permanently associated with a single account. 

SimpleLogin doesn't provide temporary emails. Any generated address is permanently associated with that account. Removing or recycling addresses is not a feature. An address will always be associated with a single account. 


As others have said, **Proton Mail and other similar legitimate emails services should absolutely not be added to this list**. 

Adding SimpleLogin will only frustrate the privacy conscious since it is not a temporary email service, and doesn't act like a temporary email service. 

Temporary email addresses will circulate through many users. SimpleLogin does not allow this.

**Comment by Zoarial94** [comment]
> > Adding SimpleLogin will only frustrate the privacy conscious since it is not a temporary email service, and doesn't act like a temporary email service.
> 
> Again, you can make infinite aliases and they'll all be directed to your mail. You can then delete that alias and create a new one. It's like you haven't even tried to use SimpleLogin.

I would love for you to help me to where the delete button because unfortunately I'm unable to find it. Any address I generate stays fixed to my account.

**Comment by Zoarial94** [comment]
> > > Adding SimpleLogin will only frustrate the privacy conscious since it is not a temporary email service, and doesn't act like a temporary email service.
> > 
> > Again, you can make infinite aliases and they'll all be directed to your mail. You can then delete that alias and create a new one. It's like you haven't even tried to use SimpleLogin.
> 
> I would love for you to help me to where the delete button because unfortunately I'm unable to find it. Any address I generate stays fixed to my account. 

While this doesn't help my case, apparently you can delete aliases (although it is discouraged). In the Android app, this functionality is not present. 

Verifying users through email is a low bar anyway. Eliminating the services which market themselves as temporary email services should be the real focus.

**Comment by yrahul3910** [comment]
> >   Eliminating the services which market themselves as temporary email services should be the real focus.
>
> I wholeheartedly agree.

Let's add in GSuite domains, then! It's as prominently marketed as Proton does. [Link](https://support.google.com/a/answer/33327?hl=en)

**Comment by jodiecunningham** [comment]
> Right. Let's also block ALL GSuite domains, shall we? They [also support email aliases](https://support.google.com/a/answer/33327?hl=en). And Outlook. And iCloud, since my iPhone defaults to aliases for new services.
> 
> From the Google Workspace docs:
> 
> > You need a temporary email address—Use an alias to sign up for a particular service or to run a temporary project.

I found you are limited to 30 aliases on an account, 100 at domain. However, the recipient map is not subject to such a limit, and of course, wildcard forwarding works.

**Comment by BetaPictoris** [comment]
This seems as though it is targeted towards Polish (`o2.pl`, `w2.pl`), Russian (`rambler.ru`), German (`tutanota.com`), and other services which are not standard outside of the US (where iCloud Mail, Gmail, and Outlook all dominate the market).  

I'd also like to point out that Skiff Mail, Proton Mail, and Tuta Mail (and I'm sure many others on this list) all, at least to the best of my knowledge, have well designed anti-abuse systems in place and blocking them for be "fake/temp emails" doesn't really make sense.  If a system administrator is having a problem with a user of said services they should be able to do what we've been doing forever, which is using those services' anti-fraud/abuse systems.  For the most part if you can prove that a user is abusing their mail service they'll work with you, most mail hosts don't want to be abused. 

For me a large number of these domains don't seem to fit in the scope of the project, but that's up to @7c to decide.

**Comment by GalacticHypernova** [comment]
> The people and community around this project are already doing this work

You're getting it mixed up. This is not eliminating the services, this is allowing that same email verification process to filter those out on the website level, but it doesn't take care of it service-level. Eliminating those services would mean actually getting them out of service. At least that's my definition of eliminating those services.


> Please don't mistake it for anything else.

I do appreciate you saying this, but I also know that there's a way to say those things. There's a way to express diagreement, and I'm not talking about you right now, I'm talking about those who thought it would be a compelling argument to say I have to be less than 13 and that I have nothing going on in my life, because that will do nothing besides make them look ridiculous that they're trying to take it to personal lines when I keep my respect and disagreement on a professional level only. And it definitely won't make me want to remove them from the list, but quite the opposite. This entire raid has not done anything besides show me that the same people who complain I'm not showing empathy decide to try and be as offensive as possible towards me. You want me to reconsider my decision? Talk to me about it maturely. I'm all for feedback and criticism, but only when it's constructive. If it's destructive there is no use for it so I will simply ignore it.

**Comment by roelofr** [comment]
> I'd also like to point out that Skiff Mail, Proton Mail, and Tuta Mail (and I'm sure many others on this list) all, at least to the best of my knowledge, have well designed anti-abuse systems in place and blocking them for be "fake/temp emails" doesn't really make sense. 

Separately from the anti-abuse systems, these mail providers are all a full, user-driven provider in their own right. 

They're completely equivalent to Gmail, Outlook, Yahoo, and your ISP's mailboxes. Adding these to a popular blocklist will just drive actual users away from your site.

Proton has been my mail client for a little under ten years now, I'd like to keep using it and not be forced to use my spam-ridden Gmail just because someone mistook an actual mail client for a burner-email client.

**Comment by AidenIsik** [comment]
Additionally, there are legitimate uses for a + in an email domain. Namely, tracking who sells your data to who, and setting up inbox filters to organise emails from different organisations, as they all get redirected to the base email.

**Comment by lgutter** [comment]
According to [fakefilter#9](https://github.com/7c/fakefilter/issues/9), multiple domains listed in this issue are already in the whitelist. (`protonmail.com` is one of them)
And those domains are presumably there since these are in fact legitimate email providers, not temporary / fake email services.

So @GalacticHypernova thank you for the effort you put into contributing, but I think this issue will need to be rewritten to either exclude whitelisted domains (and perhaps other domains that should be whitelisted but are not yet), or include very strong reasoning why the whitelisted domains should in fact be filtered.

**Comment by GalacticHypernova** [comment]
> Just admit you are wrong and move on with your life instead of being incapable and having to resort to "looking into it" with

It's not me who suggested that, someone suggested me it and I said I'll look into it. You can blame the person who asked me to block it, I quite literally just did whatever they recommended.

**Comment by michjackson** [comment]
> > Just admit you are wrong and move on with your life instead of being incapable and having to resort to "looking into it" with
> 
> It's not me who suggested that, someone suggested me it and I said I'll look into it. You can blame the person who asked me to block it, I quite literally just did whatever they recommended.

So now I see that above you've flagged gmail from multiple temp email providers, why isn't it being added to the list?

**Comment by yrahul3910** [comment]
> > > Just admit you are wrong and move on with your life instead of being incapable and having to resort to "looking into it" with
> > 
> > 
> > It's not me who suggested that, someone suggested me it and I said I'll look into it. You can blame the person who asked me to block it, I quite literally just did whatever they recommended.
> 
> So now I see that above you've flagged gmail from multiple temp email providers, why isn't it being added to the list?

I've repeatedly brought up GSuite and Apple Relay, and others Outlook, and OP has simply ignored them because they're possibly aware it's a ridiculous idea.

**Comment by Zoarial94** [comment]
> You're getting it mixed up. This is not eliminating the services, this is allowing that same email verification process to filter those out on the website level, but it doesn't take care of it service-level. Eliminating those services would mean actually getting them out of service. At least that's my definition of eliminating those services.

You're correct. I meant my comment in the context of this project. Poor work choice from me (It's too late for me to be up). 


> > Please don't mistake it for anything else.
> 
> I do appreciate you saying this, but I also know that there's a way to say those things. There's a way to express diagreement...

I won't excuse or justify those comments, but I think I can understand and explain them. To me and others alike, adding Proton Mail to this list is akin to adding Gmail for those that use Gmail. I hope you can imagine the kinds comments you would get for suggesting that Gmail should be blacklisted from creating accounts.

Whether jerks use Proton Mail or not doesn't justify blacklisting it for everyone. 

I wish everyone could give the benefit of the doubt and try to see through WHY you might have suggested what you have. Unfortunately any kind of visibility on the internet will lead to trolls. (For what it's worth I found this through Lemmy).

**Comment by GalacticHypernova** [comment]
> I've repeatedly brought up GSuite and Apple Relay, and others Outlook, and OP has simply ignored them because they're possibly aware it's a ridiculous idea.

Not quite ignored, more like didn't get to refer to it yet because of all the other people who just try to crucify me for stating my opinion.

**Comment by Techognito** [comment]
> Additionally, there are legitimate uses for a + in an email domain. Namely, tracking who sells your data to who, and setting up inbox filters to organise emails from different organisations, as they all get redirected to the base email.

Or for security reasons, figuring out what service got hacked, and preventing your user from being used across multiple services.

**Comment by yrahul3910** [comment]
OP, you've already forked this repo anyway. Surely you could just add these domains to your list and use that filter?

**Comment by Vugugftdyguv** [comment]
@GalacticHypernova what will you say about [this?](https://smailpro.com/advanced)

**Comment by Vugugftdyguv** [comment]
> > @GalacticHypernova what will you say about [this?](https://smailpro.com/advanced)
> 
> I will say thank you for reporting this, and I will also add it to the list!

You will add gmail? It is fake gmail generator…

**Comment by michjackson** [comment]
> > @GalacticHypernova what will you say about [this?](https://smailpro.com/advanced)
> 
> I will say thank you for reporting this, and I will also add it to the list!

You're still ignoring the simple comparison I asked of you. Why are you parsing out gmail addresses and commenting how hard it will be to distinguish of an address is legitimate or not on gmail but you're fine with blanket adding other legitimate mail domains?

**Comment by Vugugftdyguv** [comment]
@GalacticHypernova, please read [this article](https://support.microsoft.com/en-us/office/add-or-remove-an-email-alias-in-outlook-com-459b1989-356d-40fa-a689-8f285b13f1f2)(official from outlook). You can create many one time aliases on outlook.

**Comment by GalacticHypernova** [comment]
> you're fine with blanket adding other legitimate mail domains

This part isn't really true, and I apologize if I made it seem that way. If I find other temp email providers that provide proton accounts, I could mention it when adding it to the list and will try to make proper filtering to distinguish from legitimate users.

**Comment by michjackson** [comment]
> I am pleased to hear that you take anti-abuse seriously, but I've seen spikes in temporary email usage from SL's domains in various places online, which led to major abuse, so it is very much problematic, which is why I would appreciate if you could implement some anti-abuse mechanisms. 

@GalacticHypernova how much use of gmail have you seen an why isn't it blocked?

**Comment by GalacticHypernova** [comment]
It's always amusing to see the new issues being created with the same copy pasted message, it just proves my point further. All of these trolls that are just trying to pick up fights.

**Comment by GalacticHypernova** [comment]
> > I am pleased to hear that you take anti-abuse seriously, but I've seen spikes in temporary email usage from SL's domains in various places online, which led to major abuse, so it is very much problematic, which is why I would appreciate if you could implement some anti-abuse mechanisms.
> 
> @GalacticHypernova how much use of gmail have you seen an why isn't it blocked?

Not as much as from SL

**Comment by michjackson** [comment]
> > > I am pleased to hear that you take anti-abuse seriously, but I've seen spikes in temporary email usage from SL's domains in various places online, which led to major abuse, so it is very much problematic, which is why I would appreciate if you could implement some anti-abuse mechanisms.
> > 
> > 
> > @GalacticHypernova how much use of gmail have you seen an why isn't it blocked?
> 
> Not as much as from SL

Ok so what arbitrary level do you have to see that you decide a legitimate domain is now fake?

Do people using this project expect that you will add legitimate domains just because you've seen a "spike in abuse" on the domain?

**Comment by GalacticHypernova** [comment]
> Do people using this project expect that you will add legitimate domains just because you've seen a "spike in abuse" on the domain?

Don't forget this package doesn't automatically block the domains, it gives insight to the web operators to decide what they want to do with it, they can choose to block it or not.

**Comment by AidenIsik** [comment]
> Don't forget this package doesn't automatically block the domains, it gives insight to the web operators to decide what they want to do with it, they can choose to block it or not. It's literally the same as just checking the registering email manually.

If it really won't lead to the likes of Proton being blocked, then why exclude Gmail? I mean, it only gives insights, right?

**Comment by Zoarial94** [comment]
> I also [created the thread on PrivacyGuides](https://discuss.privacyguides.net/t/community-help-needed-fakefilter-want-to-block-all-privacy-domains/15380)

> On this thread 7c/fakefilter#73 1 some random user asks to block ProtonMail and SimpleLogin, as well as Skiff and Tuta email domains.

I really don't appreciate the wording in your post. The abrasive and disrespectful attitude you're taking in this thread isn't helping to resolve the issue. Trying to add to the dog pile isn't helpful to anyone. 

I think it's time to lock the thread. 

The author appears to want to do the right thing, and at this point you are no longer contributing to a meaningful conversation.

**Comment by michjackson** [comment]
Lock the thread now that the hypocrisy is on full display. The usual nonsense from people who can't admit they're wrong.

**Comment by GalacticHypernova** [comment]
> I think it's time to lock the thread.

I think so too. This is getting ridiculous. I'll discuss this with 7c and we'll see how to progress from there.

**Comment by michjackson** [comment]
> > I think it's time to lock the thread.
> 
> I think so too. This is getting ridiculous. I'll discuss this with 7c and we'll see how to progress from there.

"Oh no I've been caught out by my own words, time to end discussion, you're all so bad and mean to me!"

**Comment by michjackson** [comment]
Add Gmail to the list or admit you're hacks, thanks and good night

**Comment by michjackson** [comment]
> @michjackson you have no right to speak here, I can insult you right back x10 more serious but I choose to have manners. I don't blame you for not growing up to be a responsible human being because I know it's something to be learnt through parenting and it's not something you're inherently at fault for, but the initial amusement I got from you and your fellows has now turned into just annoyance, more than anything. So if you don't want to contribute to a conversation, I suggest you stop responding.

Let me know when you can answer a clear question which was posed about 20 comments back now without deflecting. You've been proven to be arbitrarily messing with legitimate mail providers and then acting like you're a victim when people call you out on it.

**Comment by GalacticHypernova** [comment]
@yrahul3910 It's honestly comedic how I'm being hit with negativity when I mention that fights aren't productive and that I want to lock the thread to filter out all the toxicity, but when you mention it it gets accepted with no issues 😆 Just shows how immature and two-sided this "community" is.

And to answer your question, which I already have beforehand but it got lost in this flood of comments, the reason I added proton and not gmail is because the abuse I noticed was orders of magnitudes lower from gmail. That doesn't mean gmail doesn't have its abuse, and I never said it was a clean domain, but the filtering that needs to be done for gmail is too insignificant to be added into this package, and it's more like edge cases that the site developers who really care about integrity would need to implement on their own.

**Comment by michjackson** [comment]
> @yrahul3910 It's honestly comedic how I'm being hit with negativity when I mention that fights aren't productive and that I want to lock the thread to filter out all the toxicity, but when you mention it it gets accepted with no issues 😆 Just shows how immature and two-sided this "community" is.
> 
> And to answer your question, which I already have beforehand but it got lost in this flood of comments, the reason I added proton and not gmail is because the abuse I noticed was orders of magnitude lower from gmail. That doesn't mean gmail doesn't have its abuse, and I never said it was a clean mail service, but the filtering that needs to be done for gmail is too insignificant to be added into this package, and it's more like edge cases that the site developers who really care about integrity would need to implement on their own.

" This is a request to add SL, among others (like protonmail which you reminded me to add), to the list as it is indeed a fake email provider. "

Why did you add protonmail which you apparently didn't even see in your statistics if this was based on level of abuse observed? If protonmail approached the same threshold that you saw from SL domains why did you not add it initially and claim you were reminded to add it? Now that gmail has been clearly shown to be a fake/temp mail provider why won't you add it as well? The level of abuse from protonmail and gmail must be similar if neither made the initial cut based on level of abuse.

**Comment by GalacticHypernova** [comment]
> Why did you add protonmail which you apparently didn't even see in your statistics if this was based on level of abuse observed?

I did see it in my statistics, I physically forgot to add it to the list, which is why I thanked them for reminding me to do it.

**Comment by michjackson** [comment]
> > Why did you add protonmail which you apparently didn't even see in your statistics if this was based on level of abuse observed?
> 
> I did see it in my statistics, I physically forgot to add it to the list, which is why I thanked them for reminding me to do it. I'm a person too, I can forget things, especially when dealing with such a big count of mail providers, and there's nothing wrong with that.

You said you saw gmail in your statistics, why don't you add it now?

**Comment by GalacticHypernova** [comment]
> Even if something's true, you dont have to bring it up.

I'm just saying, it's hard not to reference it completely when a group of ~7 people try to sabotage everything I do because of a little dispute. As I said, if I encounter destructive criticism I will simply ignore it, but I also do want to help others, which in this case would involve helping people grow to be more respectful. I just like to help, it's part of who I am.

**Comment by GalacticHypernova** [comment]
> You said you saw gmail in your statistics, why don't you add it now?

As I said, "the reason I added proton and not gmail is because the abuse I noticed was orders of magnitude lower from gmail. That doesn't mean gmail doesn't have its abuse, and I never said it was a clean mail service, but the filtering that needs to be done for gmail is too insignificant to be added into this package, and it's more like edge cases that the site developers who really care about integrity would need to implement on their own."

**Comment by GalacticHypernova** [comment]
@AidenIsik and shame on you for behaving so immaturely over an opinion that differs from yours. Now could you please stop responding if you have nothing of value to add?

**Comment by AidenIsik** [comment]
I can respect your preference until you start to push it onto filter lists.

What you're trying to do now is prop up unethical monopolies.

**Comment by Zoarial94** [comment]
> > Even if something's true, you dont have to bring it up.
> 
> I'm just saying, it's hard not to reference it at all when a group of ~7 people try to sabotage everything I do because of a little dispute. As I said, if I encounter destructive criticism I will simply ignore it, but I also do want to help others, which in this case would involve helping people grow to be more respectful. I just like to help, it's part of who I am.

It's not "just saying", you're not ignoring it by bringing something back up, and you're not being respectful by making those remarks. 

There's a time and place for it, but it's not here and now. You're throwing fuel on the fire. 

It's certainly hard, but if you really stuck to those words then a few people would've left by now.

**Comment by AidenIsik** [comment]
Yeah a lot of these are, the issue author doesn't care lol

**Comment by michjackson** [comment]
> > You said you saw gmail in your statistics, why don't you add it now?
> 
> As I said, "the reason I added proton and not gmail is because the abuse I noticed was orders of magnitude lower from gmail. That doesn't mean gmail doesn't have its abuse, and I never said it was a clean mail service, but the filtering that needs to be done for gmail is too insignificant to be added into this package, and it's more like edge cases that the site developers who really care about integrity would need to implement on their own."

Then it sounds like your decisions are arbitrary and based on your own statistics rather than a larger understanding of what you are proposing. This project claims to filter fake/temp emails. It does not claim to filter any provider that could provide a fake/temp email address and which has shown up in a random user's statistics of abusive domains impacting them which they then arbitrarily select some from the list based on volume.

Every mail provider in the world could be used for a temp/fake email address. The distribution of a given site owner's impact from domains would vary dramatically. You're basically trying to custom fit this solution to your website while ignoring the larger impact to users of these mail providers and also how other providers meeting the same criteria, like gmail or outlook, are being left off the list.

From a development standpoint your PR is flawed because of this.

**Comment by GalacticHypernova** [comment]
@Zoarial94 
Well, it's like the other guy that expects me to show empathy while trying to be as offensive as possible. I would be more than happy to respect people if they respect me, but if they don't then what good would that do? (For reference, please [view](https://github.com/7c/fakefilter/issues/73#issuecomment-1838097339))

However, you are right. This truly isn't worth my time, as amusing as it is. I suppose I got a little too invested in this because I don't like being disrespected, but I guess that's true for everyone. I can guarantee people will talk trash about me if I end the discussion as I said, but at this point I realize nothing I could do would prevent more negativity from reaching this. And with that, I noticed you mentioned it was late, so thank you and have a great night.

**Comment by GalacticHypernova** [comment]
> The aliases are not shared with other users and remain perpetually active until the user deletes them.

As I was saying, it's the aliases that are the issue. It's the fact you can endlessly delete them and create new ones, which essentially act as a bunch of disposable email addresses, which this project is also aimed at detecting. I never said it was a fake email service.

**Comment by TheSilkky** [comment]
> @Zoarial94 Well, it's like the other guy that expects me to show empathy while trying to be as offensive as possible. I would be more than willing to respect people if they respect me, but if they don't then what good would that do? (For reference, please [view](https://github.com/7c/fakefilter/issues/73#issuecomment-1838097339)) Just how you made a snarky comment about the delete button, I'm allowed to make a snarky comment about the fact that the way this community operates is sub-optimal.
> 
> > It's certainly hard, but if you really stuck to those words then a few people would've left by now.
> 
> I'm afraid doing so would just result in more users coming here to hate on me like the guy that asked 7c to ban me from contributing due to me being a "teenager troll".
> 
> However, you are right. This truly isn't worth my time, as amusing as it is. I suppose I got a little too invested in this because I don't like being disrespected. I can guarantee people will talk trash about me if I end the discussion as I said, but at this point I realize nothing I could do would prevent more negativity from reaching this. And with that, I noticed you mentioned it was late, so thank you and have a great night. (And yes, I did not say anything disrespectful in this message, because I want to end off on a good note)

Stop "civility" baiting, nobody owes you respect. Especially when you're refusing to listen to anything that anyone says on a change that could potentially effect millions of legitimate users. Go fuck yourself cunt.

**Comment by dngray** [comment]
As for adding mailbox.com that's kind of a joke, because it's a paid service with no free subscription.

**Comment by Zoarial94** [comment]
> > That doesn't mean we all share or agree with your opinion about the company.
> 
> I fully understand it, and I don't expect everyone to agree with anything I say obviously. In fact, I love it when people are contradicting me and make me explain my point of view, because it allows me to express my opinions. But accepting is different from agreeing. This could've been resolved vastly different had the negativity not reached the issue.

You're right. I also tend to ramble. You unknowingly poked a large bear, and that can be forgive. When you continue to poke the smaller bears, that's when things get worse. 

I'm not telling you not to defend yourself, but wait until the time is right to bring up specifics.

**Comment by lgutter** [comment]
> > The aliases are not shared with other users and remain perpetually active until the user deletes them.
> 
> As I was saying, it's not the aliases that are the issue. It's the fact you can endlessly delete them and create new ones, which essentially act as a bunch of disposable email addresses, which this project is also aimed at detecting. I never said it was a fake email service. What many here forgot is that this isn't about strictly fake emails, it's about disposable ones as well, and having an email you can endlessly delete and re-create is the definition of a disposable email address, because you can just dispose of it.

I mean no disrespect, but it won't help your case to say "I never said it was a fake email service" when you said the following in this same issue:
> ... This is a request to add SL, among others (like protonmail which you reminded me to add), to the list as it is indeed a fake email provider. ...

https://github.com/7c/fakefilter/issues/73#issuecomment-1785250450

**Comment by dngray** [comment]
I wouldn't be adding Tutanota, Skiff or any other "free email providers", otherwise this filter will simply become "**don't sign up with anything but custom domain list**". Most legitimate users in the world don't register their own domain to use with email (I know crazy right?), so doing so makes the whole project pointless if you mingle any free provider in there.

Spammers can still register domains stolen ID anyway, and that would result in it being totally useless. A lot of the spam I get is from some domain with no A records anyway, that was clearly registered for the purpose of spamming, either that or some box that a spammer has gained access to (and is sending email from someone else's domain). I believe you can probably buy access to these systems on the darknet in sizable numbers if you're a spammer.

On a side note, Tutanota accounts become inactive if not in use and users have to pay to reclaim those logins.  That means it would be entirely possible to send out a verification email in 6-12 months and then close the account if the user never clicks the link.

**Comment by dngray** [comment]
> I meant disposable

Except that list includes providers which aren't disposable, and are quite likely to be a user's primary email address.

**Comment by GalacticHypernova** [comment]
> Except that list includes providers which aren't disposable

As I was saying, "I can try to re-run my observations and see whether strict enforcement is still needed compared to the first time I added them", but from what I had gathered at the time of the tests (which was quite a bit ago, it's been on the list for a while) they did qualify as disposables.

**Comment by dngray** [comment]
> Do you know by any chance after how long?

it says it on their page https://tuta.com/support/#inactive-accounts

**Comment by AidenIsik** [comment]
People have been telling you this whole time that proton, tutanota, etc are not disposable.

**Comment by GalacticHypernova** [comment]
> > Do you know by any chance after how long?
> 
> it says it on their page https://tuta.com/support/#inactive-accounts

I see. Thank you!

**Comment by TheSilkky** [comment]
> > > Do you know by any chance after how long?
> > 
> > 
> > it says it on their page https://tuta.com/support/#inactive-accounts
> 
> I see. Thank you! I could try to re-consider tutanota.



> > > Do you know by any chance after how long?
> > 
> > 
> > it says it on their page https://tuta.com/support/#inactive-accounts
> 
> I see. Thank you! I could try to re-evaluate and re-consider tutanota.

Does everyone need to spoon feed you information for you to listen, or are you actually capable of doing research yourself?

**Comment by dngray** [comment]
On a side note google has [only started to delete inactive accounts](https://blog.google/technology/safety-security/updating-our-inactive-account-policies/) that haven't had a login for 2 years since gmail opened in 2003.

Protonmail also has an [innactive account](https://proton.me/support/inactive-accounts) policy as does [mailbox.org](https://kb.mailbox.org/service-desk/account-article/how-to-recover-a-deactivated-account).

[Skiff never deletes](https://www.reddit.com/r/Skiff/comments/10826u4/inactivity/) accounts but that will probably happen in the future.

**Comment by GalacticHypernova** [comment]
> probably happen in the future.

Looking forward to it, because I believe the service should prevent abuse on the service-level, rather than website-level with filtering.

**Comment by TheSilkky** [comment]
> > probably happen in the future.
> 
> Looking forward to it, because I believe the service should prevent abuse on the service-level, rather than website-level with filtering.

You obviously couldn't give a single fuck. Considering the fact that you don't even know the policies of the providers you're trying to get black listed.

**Comment by dngray** [comment]
Like i said, I have never seen spam from any of those accounts, spammers just hijack domains or boxes where they can buy credentials in bulk. Spammers don't like filling out captchas.

**Comment by TheSilkky** [comment]
> > I have never seen spam from any of those domains,
> 
> Well, you're lucky haha. My tests showed an noncancellable amount of spam from those domains, which the only reason I added those. I'm not talking about single digit or even double digit amount of incidents, I'm talking triple digit to quadruple digits.

Stop spewing bullshit and close this pr.

**Comment by dngray** [comment]
> My tests showed an noncancellable amount of spam from those domains

I am curious to know what "tests" you did, or what services you personally operate where you would even have access to that kind of data. Press D to doubt.

**Comment by TheSilkky** [comment]
> > I am curious to know what "tests" you did
> 
> I can't go into much details, but if you'd like I can explain it to you privately. I personally don't operate any super secret services but I can see incidents, thanks to the existence of the internet where a lot of those incidents are actually reported, as well as visual observations.

This just confirms that these "tests" are completely made up. Close this issue and fuck off.

**Comment by TheSilkky** [comment]
> > So basically you read it on the internet somewhere....
> 
> Not quite. While a lot of these are from reputable reports, a pretty decent amount is also from my own tests, because of course you can't take everything the internet says seriously without verifying it on your own.

Translation: ChatGPT told me

**Comment by bartbutler** [comment]
> > I am curious to know what "tests" you did
> 
> I can't go into much details, but if you'd like I can explain it to you privately. I personally don't operate any super secret services but I can see incidents, mainly thanks to the existence of the internet where a lot of those incidents are actually reported, as well as visual observations.

As this kind of abuse is a core threat to our business, we talk to a lot of people who run large services in the wild to know how we are doing and adjust accordingly. That said, our methods for anti-abuse are proactive, not reactive.

Don't get me wrong, abuse is a real problem, and smaller sites sometimes do sometimes get targeted with a tranche of stolen emails from one provider or another which is a large volume for them but not for other services, but we are really quite good at this. And we are **not** a disposable email service, full stop.

**Comment by 7c** [comment]
Dear Contributors,

We value your suggestions for expanding our list of disposable email providers. Your input is crucial in enhancing our tool's capabilities.

### Decision on Gmail and ProtonMail Inclusion
After thorough evaluation, we have resolved **not** to include Gmail and ProtonMail in our list. Our rationale is based on the following technical and operational considerations:

1. **Reputation and Reliability**  
   - **Gmail and ProtonMail**: Established, reputable providers with a high trust level for personal and professional communication.
   - **Distinction**: Unlike typical disposable email services, they offer long-term, reliable email solutions.

2. **Active Abuse and Spam Prevention Mechanisms**  
   - **Effective Systems**: Both providers have robust mechanisms to detect and mitigate abuse and spam.
   - **Proactive Monitoring**: Ensures a secure email environment, reducing the prevalence of malicious activities.

3. **Commercial Intent of Typical Disposable Email Providers**  
   - **Focus**: Targeting providers driven by ad revenue, facilitating spam/abuse.
   - **Gmail and ProtonMail's Model**: User-centric, not primarily ad-driven.

4. **Domain Limitations**  
   - **Effectiveness**: Limited domain offerings by Gmail and ProtonMail make them less susceptible to misuse.
   - **Strategy**: Focusing on providers with extensive, rotating domain lists for more impactful filtering.

5. **Individual User Accountability**  
   - **Accountability Measures**: Both services have mechanisms to penalize users violating terms, decreasing misuse risks.

### Summary and Next Steps
Including Gmail and ProtonMail does not align with our criteria for identifying disposable email services. Our aim is to target services significantly contributing to online spam and abuse, without impacting legitimate email services. We have reviewed your list and agree on adding some providers, like internxt.com ([Reference](https://github.com/7c/fakefilter/issues/79#issuecomment-1834674141)). We will also incorporate the obvious choices from the tail of your list. We apologize for the delay in addressing this issue but intend to promptly resolve it by focusing on the most impactful additions.

**Comment by dngray** [comment]
The problem is this issue literally includes  dozen other highly reputable providers and services, making it useless as a whole.

It also treats many of the services as "equal" when aliasing services and free email providers (that both have captcha and login requirements) are in fact not temporary email services (like tempmail etc).

**Comment by GalacticHypernova** [comment]
> > @7c what about the other providers? And in addition, while I agree that blocking the entire service may not be the best option, there's still a way to reliably filter through them, which I have sent you an email to so we could continue the conversation there.
> 
> We will be working on rest and decide based on the criters i mentioned above, we will for sure add the ones in this thread with obvious character of being a disposable email provider.

I would also appreciate it if you we could continue this conversation privately from where we have initiated it in #69 , away from the toxicity and negativity that has very evidently affected the flow of this issue.

**Comment by TheSilkky** [comment]
> I would also appreciate it if we could continue this conversation privately from where we have initiated it in #69 , away from the toxicity and negativity that has very evidently affected the flow of this issue.

@GalacticHypernova Or just stop being a pest and fuck off

**Comment by Brendan-D15** [comment]
Perhaps it would be better to debate what a temporary email service is vs what a forwarding email service is.

For the purpose of this list, if you broaden the definition of "temporary emails" to include the ability of a user to create forwarding email addresses (or aliases), then you may as well include Gmail, Apple/iCloud, Yahoo, and many other email services to this list which would defeat the purpose.
https://help.yahoo.com/kb/SLN28815.html

With temporary email addresses, the user typically only has access to that email for a short time. Maybe long enough to complete verification, but not long enough for the site to be able to rely on it as a means of communicating with the user into the future. 

With forwarding email services, the user does retain access to the address and can be reliably reached. This would include services such as Gmail, Apple/iCloud, Yahoo, Addy.io, Protonmail and more (as has been pointed out by others in this thread). In most cases the user can even respond using that forwarding email address (Gmail being the exception here).
See also: https://support.apple.com/guide/icloud/what-you-can-do-with-icloud-and-hide-my-email-mme38e1602db/1.0/icloud/1.0
https://support.google.com/a/answer/33327?hl=en#whatEmailAlias

The disposability of an email address should not be a considering factor as to whether or not an email is temporary. Anyone can create or delete an email alias for iCloud or Gmail at any time:
https://support.apple.com/guide/icloud/what-are-email-aliases-in-icloud-mail-mm074af79454/1.0/icloud/1.0

It seems like @GalacticHypernova is attempting to solve the question of whether a user is truly unique to a site or not. This is not a question that can be solved by relying on an email address alone.

**Comment by dngray** [comment]
Whether you give a free pass to gmail users you probably don't know who they are either if they are using email addresses that don't contain their real name which is quite common.

It's also trivial to open accounts on those providers.

**Comment by AidanWelch** [comment]
In addition to what others have said, seznam.cz is a major provider in Czech Republic and would block many Czech users.  I'm sure other domains are similarly important to other users.

**Comment by Hxyfufuf** [comment]
> > This made my day. I guess this issue can be closed, as OP has no clue of what they are speaking of.
> 
> OP is a fucking moron

I agree. He should change username for this before someone will stole it XD

**Comment by amilich** [comment]
Hey all. I'm the CEO of Skiff - we're **not** an anonymous email provider. Why did we end up on this list?

It's quite disappointing to build an email provider that has grown to millions of users, and then we've never been contacted by anyone on this thread with questions or requests to understand the issues that got us here.

I opened a PR to remove Skiff because we're not at all in the same category here.

**Comment by amilich** [comment]
Hello @GalacticHypernova - would quite appreciate the reversion on this given the info above. Happy to clarify anything needed.

**Comment by amilich** [comment]
> The problem is this issue literally includes many other highly reputable providers and services, making it useless as a whole.
> 
> It also treats many of the services as "equal" when aliasing services and free email providers (that both have captcha and login requirements) are in fact not temporary email services (like tempmail etc).

I completely agree.

**Comment by amilich** [comment]
> > I am curious to know what "tests" you did
> 
> I can't go into much details, but if you'd like I can explain it to you privately. I personally don't operate any super secret services but I can see incidents, mainly thanks to the existence of the internet where a lot of those incidents are actually reported, as well as visual observations.

I'm the CEO of Skiff. Can you share these instances with us instead of degrading the experience of signing up for products all around the web for millions of people?

The impact here is enormous. By blocking providers that are up-and-coming, but not disposable, it makes it much harder for anyone to compete with big tech, ad-supported email clients like Yahoo/Gmail.

**Comment by amilich** [comment]
> @Nikola-Test-for-Proton Are you willing to settle for a more convenient method of communication, like Discord? Emails would be a little hard to work with for a potentially prolonged discussion.

We're happy to communicate via Discord, Reddit, GitHub, email (andrew @ skiff dotcom), or any communication mechanisms.

**Comment by xe-3** [comment]
1. Proton, Skiff, Tutanota are legitimate professional e-mail providers. _These services should absolutely not be on your list_
2. Addy.io and Simplelogin are services used for many things, but not temporary mail. The primary purpose of these services is reducing spam and phishing emails, there are also some positive benefits with respect to security and privacy. If you are familiar with Apple or Mozilla's mail services, these work in the same way. _These services should not be on your list_

In my experience (having moderated a few forums over the years and run a couple websites), Gmail and a couple other major providers in the US, China, and Russia are the source of vastly more spam accounts and bot signups than any of the smaller service providers I have listed. Penalizing the small players, is both unfair and ineffective.

**Comment by nishils** [comment]
> Dear Contributors,
> 
> We value your suggestions for expanding our list of disposable email providers. Your input is crucial in enhancing our tool's capabilities.
> 
> ### Decision on Gmail and ProtonMail Inclusion
> After thorough evaluation, we have resolved **not** to include Gmail and ProtonMail in our list. Our rationale is based on the following technical and operational considerations:
> 
> 1. **Reputation and Reliability**
>    
>    * **Gmail and ProtonMail**: Established, reputable providers with a high trust level for personal and professional communication.
>    * **Distinction**: Unlike typical disposable email services, they offer long-term, reliable email solutions.
> 2. **Active Abuse and Spam Prevention Mechanisms**
>    
>    * **Effective Systems**: Both providers have robust mechanisms to detect and mitigate abuse and spam.
>    * **Proactive Monitoring**: Ensures a secure email environment, reducing the prevalence of malicious activities.
> 3. **Commercial Intent of Typical Disposable Email Providers**
>    
>    * **Focus**: Targeting providers driven by ad revenue, facilitating spam/abuse.
>    * **Gmail and ProtonMail's Model**: User-centric, not primarily ad-driven.
> 4. **Domain Limitations**
>    
>    * **Effectiveness**: Limited domain offerings by Gmail and ProtonMail make them less susceptible to misuse.
>    * **Strategy**: Focusing on providers with extensive, rotating domain lists for more impactful filtering.
> 5. **Individual User Accountability**
>    
>    * **Accountability Measures**: Both services have mechanisms to penalize users violating terms, decreasing misuse risks.
> 
> ### Summary and Next Steps
> Including Gmail and ProtonMail does not align with our criteria for identifying disposable email services. Our aim is to target services significantly contributing to online spam and abuse, without impacting legitimate email services. We have reviewed your list and agree on adding some providers, like internxt.com ([Reference](https://github.com/7c/fakefilter/issues/79#issuecomment-1834674141)). We will also incorporate the obvious choices from the tail of your list. We apologize for the delay in addressing this issue but intend to promptly resolve it by focusing on the most impactful additions.

Hi @7c,

Thanks for posting this clear reasoning. I am the Director of Security at Skiff and have worked on building much of the anti abuse at Skiff.

We actively have Skiff users that have started to use us for business, professional, and personal usage. Though Skiff is newer to the market, we shouldn't be penalized on being a new provider. We should be judged on our anti abuse systems, and we are happy to work with anyone that sees issues with skiff.com email addresses.

We have quite a few anti abuse mechanisms to prevent spamming third party registrations similar to all the larger players in this space. We also build this into our product where we can. In fact to prevent abuse of our email aliasing service, we've made it easy for website operators to cluster similar accounts via the subdomain on maskmy.id as an anti abuse protection. As a skiff user, you could claim the subdomain 7cfakefilter.maskmy.id which is only available to you. If you sign up with a ton of accounts using this subdomain, a service provider can simply block the subdomain to prevent abuse from this user. We believe that signals like this for website owners are extremely important while still balancing user privacy interests.

Skiff's commercialization is user-centric similar to Proton. We are not ad driven like many temporary email solutions. We also have limited domain offerings and ban accounts that engage in abusive behavior. In fact, many of them show up in our Discord or support channels claiming their accounts were banned.

We plan to continue to significantly invest in anti abuse capabilities for the foreseeable future including combatting mass signups on third party websites.

**Comment by merole** [comment]
**Seznam.cz** should be removed.

seznam.cz is a completely legitimate email service. In fact it is the largest web servives provider in the Czech tepublic.
They even have their own search engine, online second hand market place and map app with their own map data here in CZ and even a news website. https://en.m.wikipedia.org/wiki/Seznam.cz

Yes their service is a freemail one, but a lot of people use it as their mail email.

**Comment by ganthern** [comment]
Hello @7c, @GalacticHypernova,

I'm from the Tuta development team.

I'd like to point out that we at [tuta](www.tuta.com) (until recently tutanota) have been a privacy-focused email provider for more than a decade and that adding us to any list that's used to block signups to a significant number of services would disrupt daily operations for millions of businesses and private customers. 

We're just as committed to prevent abuse of our systems as the bigger players in this space, maybe even more so because we don't get the benefit of the double standard that can be observed in this very issue.

As one might guess, it would be bad for business were we to gain a reputation for enabling spam and mass signups, which is why we have several layers of automated abuse detection and manual approval of suspicious registrations.

So please reconsider this addition and make sure that our domains are not on this list, specifically:

tutanota.com
tutanota.de
tutamail.com
tuta.io
keemail.me
tuta.com

Thanks!

**Comment by rico-vz** [comment]
> Dear Contributors,
> 
> We value your suggestions for expanding our list of disposable email providers. Your input is crucial in enhancing our tool's capabilities.
> ### Decision on Gmail and ProtonMail Inclusion
> 
> After thorough evaluation, we have resolved **not** to include Gmail and ProtonMail in our list. Our rationale is based on the following technical and operational considerations:
> 
>     1. **Reputation and Reliability**
>        
>        * **Gmail and ProtonMail**: Established, reputable providers with a high trust level for personal and professional communication.
>        * **Distinction**: Unlike typical disposable email services, they offer long-term, reliable email solutions.
> 
>     2. **Active Abuse and Spam Prevention Mechanisms**
>        
>        * **Effective Systems**: Both providers have robust mechanisms to detect and mitigate abuse and spam.
>        * **Proactive Monitoring**: Ensures a secure email environment, reducing the prevalence of malicious activities.
> 
>     3. **Commercial Intent of Typical Disposable Email Providers**
>        
>        * **Focus**: Targeting providers driven by ad revenue, facilitating spam/abuse.
>        * **Gmail and ProtonMail's Model**: User-centric, not primarily ad-driven.
> 
>     4. **Domain Limitations**
>        
>        * **Effectiveness**: Limited domain offerings by Gmail and ProtonMail make them less susceptible to misuse.
>        * **Strategy**: Focusing on providers with extensive, rotating domain lists for more impactful filtering.
> 
>     5. **Individual User Accountability**
>        
>        * **Accountability Measures**: Both services have mechanisms to penalize users violating terms, decreasing misuse risks.
> 
> 
> ### Summary and Next Steps
> 
> Including Gmail and ProtonMail does not align with our criteria for identifying disposable email services. Our aim is to target services significantly contributing to online spam and abuse, without impacting legitimate email services. We have reviewed your list and agree on adding some providers, like internxt.com ([Reference](https://github.com/7c/fakefilter/issues/79#issuecomment-1834674141)). We will also incorporate the obvious choices from the tail of your list. We apologize for the delay in addressing this issue but intend to promptly resolve it by focusing on the most impactful additions.

I hope you guys stick to _just_ blocking fake & temporary emails as stated in your description and don't start considering forwarding services (like anonaddy & simplelogin) temporary emails. 
Forwarded addresses aren't temporary. They are as permanent as any non forwarded address.

It seems like the OP has a misunderstanding of what this solution should be for as some of the domains listed by the OP are regular, moderated email services used by plenty of individuals & businesses around the world (Skiff, Tuta, Proton, etc.) If anything, these services could be considered for the whitelist, not the other way round.

**Comment by GalacticHypernova** [comment]
> Forwarded addresses aren't temporary. They are as permanent as any non forwarded address.

How are they permanent when you can just delete aliases and create new ones? It's just like a temporary email

**Comment by GalacticHypernova** [comment]
@ganthern I appreciate you reaching out. I saw the changes you made in tuta with the anti abuse, and I do appreciate you making a nice captcha system and having an inactive account policy. It's a noticeable security improvement compared to when I first added it to the list, so I'll gladly remove it.

**Comment by rico-vz** [comment]
> > Forwarded addresses aren't temporary. They are as permanent as any non forwarded address.
> 
> How are they permanent when you can just delete aliases and create new ones? It's just like a temporary email

Anyone could also create a new email on top providers like Gmail & Outlook, delete them afterward and create new ones. SL & AA are just as permanent as those. And I'd argue those 2 services have a more approachable anti-abuse team than Google or Microsoft. 


> @ganthern I appreciate you reaching out. I saw the changes you made in tuta with the anti abuse, and I do appreciate you making a nice captcha system and having an inactive account policy. It's a noticeable security improvement compared to when I first added it to the list, so I'll gladly remove it.

I'm not affiliated with Tuta whatsoever, but I've used them in the past. These policies/measures have been a thing for long before this repo issue was created... Just like their 'on hold' function for newly created emails, [https://tuta.com/support#onhold](https://tuta.com/support#onhold).

**Comment by GalacticHypernova** [comment]
@amilich if you could invite me to a private repo of yours so I could send you my username I would very much appreciate it, and we could continue the conversation from there.

**Comment by AidanWelch** [comment]
> > > Forwarded addresses aren't temporary. They are as permanent as any non forwarded address.
> > 
> > 
> > 
> > How are they permanent when you can just delete aliases and create new ones? It's just like a temporary email
> 
> This is an extremely weak argument. You could argue the same thing with Outlook.com or personal domains that allows you to add aliases and remove them at whim. 
> 
> You aren't just adding temporarily email domains, but domains people pay for. 
> 
> The difference between forwarding services and temporary services is that forwarding services are primarily used to receive email anonymously to your inbox whereas temporary services are just that. 

You can also add aliases with Gmail, and with Google Workspace it's even easier to make disposable Google accounts
